src/components/exchange/ExchangeInputField.js

```diff
@@ -50,6 +50,7 @@ export default function ExchangeInputField({
         setAmount={setInputAmount}
         symbol={inputCurrencySymbol}
         testID={testID}
+        useCustomAndroidMask={android}
```

Comment:
If we're using it in every component on android maybe we don't need to make it as a parameter and just move platform check to a component  :)

Reply:
sure

Reply:
Actually no. It's not on every component. The native input amount also use it and I don't want to add it to those. That's why I passed the prop from the top level